lang/compiler.go

```diff
@@ -493,6 +494,12 @@ func (p *AstProgram) Start(ctx context.Context, alloc memory.Allocator) (flux.Qu
 	// execution begins.
 	deps.ExecutionOptions.ConcurrencyLimit = feature.QueryConcurrencyLimit().Int(ctx)
 
+	// Create a MutableFlagger for this program, ensuring that any changes it
+	// makes to feature flags are scoped to its own execution.
+	flagger := pkgfeature.GetFlagger(ctx)
+	mflagger := pkgfeature.NewMutableFlagger(flagger)
+	ctx = pkgfeature.Inject(ctx, mflagger)
+
```

Comment:
I'm wondering about what would happen if we accidentally had `import "debug" debug.setFeature(...)` in a stdlib file. Would any script that imported that file have that feature flag set?

Reply:
Also, I'm curious if this will work in the REPL. Do we create a new context for each line of input?

Reply:
I'll test the REPL but I don't think we need it for each new line. 

Since its a function importing won't call the function and so won't have any effect.